### DISCUSSION TITLE
Don't insert default values?

### DISCUSSION THREAD

**Discussion by nvitaterna** [prisma/prisma#9951] [Q&A] [answered]
I have my database setup for default values - though Prisma is still inserting them based on the schema. Is there any way to tell Prisma that a field has a default in the database, without having Prisma add those values to the insert query? The issue with simply removing @default is that it requires that value to be set on the insert query.

**Comment by austincrim** [comment, marked as answer]
Hi @nvitaterna, thanks for the question.

What kind of default values do you have setup in your db? Are they more complex than what `@default` can handle?
Also are you using `prisma db pull`?

Small follow-up: You might want to take a look at the [`dbgenerated` attribute](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#dbgenerated). It helps support default values that aren't currently supported in the Prisma Schema Language.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
